### PR TITLE
Update swagger parser version

### PR DESCRIPTION
**Pull request by TharmiganK**
## Purpose

> $Subject

### CODE REVIEW

File: openapi-cli/src/test/resources/ballerina-to-openapi/advance/openapi/multiple_service_01.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ErrorPayload'
+                $ref: "#/components/schemas/ErrorPayload"
```
Comment (NipunaRanasinghe) [inline openapi-cli/src/test/resources/ballerina-to-openapi/advance/openapi/multiple_service_01.yaml:30]: What's the reason for this change? What will be the behaviour with the new parser version if the contract is having single quotes for the ref schema values?

Comment (TharmiganK) [reply openapi-cli/src/test/resources/ballerina-to-openapi/advance/openapi/multiple_service_01.yaml:30]: This change is only for the schema generation from the OAS model object. The parser works for both single and double quotes.

Comment (NipunaRanasinghe) [reply openapi-cli/src/test/resources/ballerina-to-openapi/advance/openapi/multiple_service_01.yaml:30]: I see. That'll be fine then

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

File: ballerina-to-openapi/build.gradle [hunk 2]
```diff
@@ -38,6 +38,7 @@ dependencies {
         exclude group: "javax.validation", module: "validation-api"
     }
     implementation "io.swagger.core.v3:swagger-models"
+    implementation "io.swagger.parser.v3:swagger-parser-safe-url-resolver:${swaggerParserVersion}"
```
Comment (lnash94) [inline ballerina-to-openapi/build.gradle:41]: How do we use this `parser-safe-url-resolver `, is it using the swagger parser itself or do we need to use it within our implementation separately?

Comment (TharmiganK) [reply ballerina-to-openapi/build.gradle:41]: This is a dependency of `swagger-parser`, from this latest version. There were `NoClassDefFound` errors without this as a dependency

**Review by lnash94 — APPROVED** [review]
